src/component/FileManager/Explorer/GalleryView/GalleryImage.tsx

```diff
@@ -93,8 +93,11 @@ const GalleryImage = memo((props: FileBlockProps) => {
       return;
     }
 
+    // Reset to loading state before reloading thumb (e.g., after reset)
+    setImageLoading(true);
+    setThumbSrc(undefined);
     tryLoadThumbSrc();
-  }, [inView]);
+  }, [inView, file, file.metadata?.[Metadata.thumbDisabled]]);
```

Comment:
This will trigger a reload of the thumbnail after reset. But you already loaded once in https://github.com/cloudreve/frontend/pull/306/files#diff-acc4829d8b9ac4ca5b1f91063828fb9acd9704b8d210258944946090996786e1R1201

So two identical requests will be made?